style.css

```diff
@@ -0,0 +1,85 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body{
+    font-family: Arial, Helvetica, sans-serif;
+    width: 100%;
+}
+
+.container{
+    width: 400px;
+    background-color: #0e0101;
+    color: rgb(221, 210, 210);
+    margin: 0 auto;
+}
```

Comment:
- [ ] OPTIONALL: I suggest you to delete this fixed width, because it will render the document wrong if mobile screen is smaller or wider then 400 px. You can use developer tools and set it to mobile screen width to check the styles.